app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -299,13 +299,13 @@ You are not eligible for the second instalment of the IRP as the dependant visa
 
 ## How to apply for the IRP
 
-We will update this page with a link to the application form when it is live.
+Applications for the IRP are open until 31 March 2025.
 
-Before you apply for the first and second instalments of the IRP, you’ll need to check you meet the eligibility requirements.
+Apply for the International Relocation Payment.
```

Comment:
<a href="https://www.claim-additional-teaching-payment.service.gov.uk/get-a-teacher-relocation-payment/landing-page">Apply for the International Relocation Payment</a>.